### DISCUSSION TITLE
Play message order by header timestamp

### DISCUSSION THREAD

**Discussion by zhangfei56** [foxglove/community#424] [Q&A]
https://github.com/foxglove/studio/pull/3311, https://github.com/foxglove/studio/pull/3909
Our team think this is very important for debug

**Comment by qdhuxp** [comment]
I can say this is definitely an important feature for detail debug. 

There a lots of ros topics published with same TF reference timestamp, but maybe because of the some reasons, we receive some of them with latency, such as {topic1, topic2, topic3}, {topic1, topic 2}, {topic1, topic 2, topic 3} ......, currently, foxglove use receive time to render the topics ,may cause the delayed {topic3} rendered in wrong TF. 

That is why I think this feature is important.
